src/build/buildExecutor.ts

```diff
@@ -66,6 +80,35 @@ export class BuildExecutor {
         return buildResult;
     }
 
+    public startLanguageServer(input: BuildInput, buildUserToken: string) {
+        let buildParameters = this.getBuildParameters(undefined, input, buildUserToken);
+        if (buildUserToken) {
+            buildParameters.envs['DOCS_OPS_TOKEN'] = buildUserToken;
+        }
+        let command = this._binary;
+        let args = buildParameters.serveCommand;
+        let options = { env: buildParameters.envs, cwd: this._cwd };
+        let serverOptions: ServerOptions = {
+            run: {
+                command,
+                args,
+                options
+            },
+            debug: {
+                command,
+                args,
+                options
+            },
+        };
+
+        let clientOptions: LanguageClientOptions = {};
+
+        this._eventStream.post(new BuildProgress(`command: ${command}, args: ${args.join(' ')}`));

```

Comment:
how about make this more readable like the `build` since this will display on the channel?